libs/feature/dataviz/src/lib/service/data.service.ts

```diff
@@ -197,7 +197,10 @@ export class DataService {
       link.type === 'service' &&
       link.accessServiceProtocol === 'esriRest'
     ) {
-      const url = this.getDownloadUrlFromEsriRest(linkUrl, 'geojson')
```

Comment:
Maybe the `linkUrl` was not explicit enough...
Should be `linkProxifiedUrl`, could have help to know to not use it, as `getDownloadUrlFromEsriRest` also proxifies the url.

By the way, I also see that in the same function, the call of `getDownloadUrlsFromWfs` also proxifies the URL twice, so to me, the codeflow within this method is confusing.

Reply:
Yep, true I'll rename it and see if I can improve more, like removing the double proxying for WFS, which does not cause errors right now thanks to [ogc-client](https://github.com/camptocamp/ogc-client/blob/main/src/shared/http-utils.ts#L80).

Reply:
Ok, I've adapted my previous commit accordingly.